app/views/rating_questions/index.html.erb

```diff
@@ -1,7 +1,12 @@
 <div class="flash-notice"><%= flash[:notice] %></div>
+<div class="flash-errors"><%= flash[:alert] %></div>
 <%= link_to "New Question", new_rating_question_path %>
 <div class='questions'>
   <% @rating_questions.each do |question| %>
-    <%= question.title %>
+     <p id='question_<%= question.id %>'> 
+     <%= question.title %> 
+     <%= link_to "Edit Question", edit_rating_question_path(question) %> |
```

Comment:
It is possible here to use the shorter router helpers:

```
<%= link_to "Edit Question", [:edit, question] %>
<%= link_to "Delete Question", question, method: :delete, data: { confirm: 'Are you certain you want to delete this?' } %>
```

These will expand out to the right routes, as I show in [my Polymorphic Routes](https://ryanbigg.com/2018/12/polymorphic-routes) write up.